poetry.lock

```diff
@@ -1,46 +1,25 @@
-# This file is automatically @generated by Poetry and should not be changed by hand.
+# This file is automatically @generated by Poetry 1.6.1 and should not be changed by hand.
```

Comment:
Maybe we should also update poetry to 1.6.1 (in Dockerfile)?

Reply:
Fix